topic-operator/src/main/java/io/strimzi/operator/topic/KafkaStreamsTopicStore.java

```diff
@@ -19,7 +19,7 @@
  * Apicurio Registry's gRPC based Kafka Streams ReadOnlyKeyValueStore
  */
 public class KafkaStreamsTopicStore implements TopicStore {
-    private static final Logger log = LoggerFactory.getLogger(KafkaStreamsTopicStore.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(KafkaStreamsTopicStore.class);
```

Comment:
I'm confused. Sometimes you use Reconciliation logger although it is not needed. Sometimes you keep using the regular one. This is confusing.